tests/providers/azure/services/monitor/monitor_service_test.py

```diff
@@ -0,0 +1,101 @@
+from unittest import mock
+from unittest.mock import patch
+
+from prowler.providers.azure.services.monitor.monitor_service import (
+    DiagnosticSetting,
+    Monitor,
+)
+from tests.providers.azure.azure_fixtures import (
+    AZURE_SUBSCRIPTION,
+    set_mocked_azure_audit_info,
+)
+
+
+def mock_monitor_get_diagnostics_settings(_):
+    return {
+        AZURE_SUBSCRIPTION: [
+            DiagnosticSetting(
+                id="id",
+                name="name",
+                type="type",
+                event_hub_authorization_rule_id="event_hub_authorization_rule_id",
+                event_hub_name="event_hub_name",
+                metrics="metrics",
+                logs=[
+                    mock.MagicMock(enabled=True),
+                    mock.MagicMock(enabled=True),
+                    mock.MagicMock(enabled=True),
+                    mock.MagicMock(enabled=True),
+                    mock.MagicMock(enabled=False),
+                    mock.MagicMock(enabled=True),
+                    mock.MagicMock(enabled=False),
+                    mock.MagicMock(enabled=False),
```

Comment:
What is the meaning of this?

Reply:
Creating a valid object for testing my check, with the new code, its also creates the category for each one.